.travis.yml

```diff
@@ -3,13 +3,12 @@ language: php
 services:
   - docker
 
-before_script:
+install:
   - docker-compose -v
+  - docker-compose build
+  - docker-compose run --rm app composer ci
 
-install:
-  - docker-compose run --rm dev composer install
-  - docker-compose run --rm dev ./vendor/bin/phpcs --standard=psr2 --ignore=vendor -n .
-  - docker-compose run --rm dev ./vendor/bin/phpstan analyse ./src --level=max -c phpstan.neon
+script: skip
 
 php:
```

Comment:
Nemyslím si, že tohle funguje - tohle ovlivní jakou verzi PHP bude mít ten worker na travisu. Tj. buď by testy musely mít 3 různé docker image nebo se musí spustit bez dockeru - viz  https://github.com/keboola/php-component/blob/master/.travis.yml

Reply:
tj. dokonce myslím, že to nikdy nefungovalo...

Reply:
A zbytečně to pouštělo to samé 3x...

Reply:
máš pravdu, pouštělo se to samé 3x

Reply:
předělal jsem to a na PHP 5.6 se to kvůli phpstanu  nerozeběhne
https://travis-ci.org/github/keboola/php-datatypes/jobs/666400631?utm_medium=notification&utm_source=github_status

Reply:
a na 7.1 to zase neproběhne protože tam chyběly return typehinty:
https://travis-ci.org/github/keboola/php-datatypes/jobs/666400633?utm_medium=notification&utm_source=github_status

Reply:
tak php 5.6 bych vyhodil a přidal 7.2 a 7.3

Reply:
Hodil bych do pryč i 7.0 když už vyhazujem 5.6, ta už má dávno EOL. 7.1 bohužel zatím musíme nechat kvůli connection.

Reply:
Souhlas.

Reply:
Nepoužívá se to ještě někde na klientovi? (ten je furt 5.6 kvuli nějakým syrup vecem)

Reply:
tj. dávalo by mi pak smysl mít aspoň parallel lint na 5.6 (tj. že jsme omylem nepřidali něco 5.6 incompatible)